antora-ui-camel/src/css/vars.css

```diff
@@ -131,11 +131,14 @@
   --footer-height: 23rem; /* empirically corresponding current design of the foooter */
   /* dimensions */
   --navbar-height: calc(73 / var(--rem-base) * 1rem);
+  --toolbar-nav-height: calc(139.5 / var(--rem-base) * 1rem);
+  --panel-height: calc(54 / var(--rem-base) * 1rem);
```

Comment:
These are some odd values, how did we came to them?

Reply:
Alright, the toolbar nav height is introduced for the width < 1275 as we introduce the whole search design below the actual navbar including the logo and social icons. 

Now with the changes, I observed that within the menu panel height under the docs section, the panel items overflow to the footer due to the height defined. Hence, the panel height is used to prevent the overflow.

Reply:
Alright, the toolbar nav height is introduced for the width < 1275 as we introduce the whole search design below the actual navbar including the logo and social icons. 

Now with the changes, I observed that within the menu panel height under the docs section, the panel items overflow to the footer due to the height defined. Hence, the panel height is used to prevent the overflow.